hosting/learnings-from-2025.md

```diff
@@ -0,0 +1,80 @@
+# Learnings from the 2025 Hackathon
+
+All in all, this was surprisingly successful. There were some technical issues though.
+
+## Overall organization
+
+* Andrew, Pier-Luigi, Sara and Yuting did a great job at holding things together
+* Bjorn did a great job at shaking up structures and pushing things forward
+* Tobi, Lukas, Mark M, and others made technical magic possibble.
+
+## Schedule
+
+## Inter-node communication
+
+### Mattermost
+
+* Works
+* Invite more people earlier in the process
+* China uses WeChat for most communication (in Chinese, but with auto-translation)
+
+### Zoom calls during the preparation phase
+
+* Double Telcos are tiring but necessary.
+* Did anybody read the nodes / watch the recordings?
+
+### Zoom calls during the hackathon
+
+* Did not really line up with the schedules (at least at some nodes). Would need earlier planning to be incorporated into the local schedules.
+
+### GitHub and Code sharing
+
+* GitHub seems to provide a bit of a barrier for sharing code.
+* Need a concept that scales to 500 people contributing at the same time without established teams or time for learning.
```

Comment:
I see that GitHub is a hurdle for many scientists. However, there is o other "concept that scales to 500 people".

The only option I see is to train people better **before** the hackathon. In addition, one could provide better guidelines for a workflow that scales:
1. **No Jupyter notebooks**
2. Commit and merge frequently
3. Difference between libraries, examples, and a dump

Reply:
And, once again, this comes back to defining the purpose of code sharing beforehand.

For this hackathon, we explicitly agreed that the GitHub repos are landfills in which participants can dump untested and unreviewed code. From what I have seen, this goal has been achieved.

But maybe we want to refine this goal for future hackathons 😉